modules/schematrons/pages/index.adoc

```diff
@@ -0,0 +1,93 @@
+----
+<rule context="/*/cac:ProcurementProjectLot[cbc:ID/@schemeName='Lot']/cac:ProcurementProject[$noticeSubType = '16']"> <!--1-->
+    ...
+    <assert id="BR-BT-00024-0178" role="ERROR" diagnostics="BT-24-Lot" test="count(cbc:Description) &gt; 0"> <!--2-->
+        rule|text|BR-BT-00024-0178 <!--3-->
+    </assert>
+    ...
+</rule>
+----
+<1> The rule element defines a set of assertions that will be executed at each location corresponding to the XPath expression in the context attribute.
+<2> The `assert` element define a single assertion with related information
+<3> The message used when the assertion is not true. This is the identifier of a translation text.
+
+The information in the `assert` element is used to provide details in the validation report. 
+
+== Validation report
+
+The validation report indicates the rules that were executed, and gives detailed information for each failed assertion.
+
+[source,xml]
+----
+<svrl:fired-rule context="/*/cac:TenderingProcess[$noticeSubType = '16']"/> <!--1-->
+...
+<svrl:failed-assert id="BR-BT-00024-0178"
+        location="/cn:ContractNotice/cac:ProcurementProjectLot[2]/cac:ProcurementProject"
+        test="count(cbc:Description) > 0"
+        role="ERROR"> <!--2-->
+    <svrl:text>rule|text|BR-BT-00024-0178</svrl:text> <!--3-->
+    <svrl:diagnostic-reference diagnostic="BT-24-Lot" see="field:BT-24-Lot"> <!--4-->
+        <svrl:text>cbc:Description</svrl:text>
+    </svrl:diagnostic-reference>
+</svrl:failed-assert>
+----
+<1> Indicates that the context for a specific rule was found in the notice
+<2> An `assert` failed because the `test` evaluated to `false`
+<3> The message for the failed assertion.
+<4> Additional information on the failure.
+
+NOTE: If you execute the Schematron validation yourself, the message will be the identifier of the message, as shown above. But if you use CVS, the validation report will contain the text of the message in the message indicated as a parameter of your request to CVS.
+
+The attributes of the `failed-assert` element provide specific information:
+
+[horizontal]
+`id`:: The identifier of the failed assertion
+`location`:: The exact location that was matched by the rule context, as an absolute XPath.
+`test`:: The XPath expression of the check that failed.
+`role`:: The severity of the failure, either `ERROR` or `WARN`.
+`flag`:: A specific characteristic of the failed assertion. The value `LAWFULNESS` indicates that the failure indicates the notice might not be suitable to be published.
+
+When relevant, additional information is provided via the `diagnostic-reference` element:
+
+[horizontal]
+`diagnostic`:: This attribute contains the identifier of the diagnostic information. This should not be used to extract information from the report.
+`see`:: This attribute contains the identifier of the node or field that is targetted by the failed assertion. The value starts either with `node:` or `field:` to distinguish the 2 types of identifiers.
```

Comment:
targetted should be targeted

Reply:
This has been corrected in Paul's commit.